test/runtests.jl

```diff
@@ -140,10 +140,10 @@ const sf_params = SurfaceFluxes.Parameters.SurfaceFluxesParameters{
 
             sfc_output = SF.surface_conditions(sf_params, sc)
             @test SF.obukhov_length(sfc_output) > FT(0)
-            @test sign(SF.non_zero_lmo(1.0)) == 1
-            @test sign(SF.non_zero_lmo(-1.0)) == -1
-            @test sign(SF.non_zero_lmo(-0.0)) == 1
-            @test sign(SF.non_zero_lmo(0.0)) == 1
+            @test sign(SF.non_zero(1.0)) == 1
```

Comment:
I ma not sure these are good values for `b_star` (1,-1) they might be a bit large

Reply:
This is not for bstar. It is to test the `non_zero` function in general. It was called `non_zero_lmo` before when we only protect lmo then. I believe these lines are to test the sign but yes we could add some more physical values.